conf/messages

```diff
@@ -1685,7 +1684,7 @@ obi.summary.headingPlural                  = You have added {0} business involve
 obi.summary.subHeading                     = Do you need to add another business?
 obi.summary.hint                           = You must tell us about any other businesses the partners or directors are a sole trader, partner or director of in the United Kingdom or Isle of Man.
 validation.obi.summary.error               = Select yes if you need to add another business
-obi.summary.max.limit.reached              = We do not need any more than {0} businesses
+obi.summary.max.limit.reached              = We do not need any more than 10 businesses
```

Comment:
I see why you've removed this, but I think the correct solution should be to add the OBI limit to config and pass it through in the controller. That'll fix the original issue and also give us the ability to change the limit on the fly without rebuilding (we'd only need to redeploy with new config)